src/App.js

```diff
@@ -45,14 +46,27 @@ class App extends Component {
   filterVideoByType = (dataVideos) => {
     let trailerVideo = dataVideos.filter(video => video.type ===  "Trailer")
     this.setState({
-      videos:[...this.state.videos, ... trailerVideo]
+      videos:[...this.state.videos, ...trailerVideo]
     })   
   }
 
   returnHome = () => {
-    this.setState( {selectedMovie: null, videos: []} )
+    this.setState( {selectedMovie: null, videos: [], filteredMovies: []} )
   } 
 
+  filterMovies = (searchInput) => {
+    let lowerCaseInput = searchInput.toLowerCase();
+    const matchingMovieTitles = this.state.movies.filter(movie => {
```

Comment:
I really like the name you gave here to this const! 
 Not sure if we should use a let insted here? 
Just a note for us to think together about it 🤔